app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/CollectionHeaderViewHolder.kt

```diff
@@ -4,14 +4,66 @@
 
 package org.mozilla.fenix.home.sessioncontrol.viewholders
 
-import android.view.View
-import androidx.recyclerview.widget.RecyclerView
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.LifecycleOwner
 import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.ComposeViewHolder
+import org.mozilla.fenix.compose.SectionHeader
+import org.mozilla.fenix.theme.FirefoxTheme
 
 class CollectionHeaderViewHolder(
-    view: View
-) : RecyclerView.ViewHolder(view) {
+    composeView: ComposeView,
+    viewLifecycleOwner: LifecycleOwner,
+) : ComposeViewHolder(composeView, viewLifecycleOwner) {
     companion object {
         const val LAYOUT_ID = R.layout.collection_header
     }
+
+    @Composable
+    override fun Content() {
+        Column {
+            Spacer(modifier = Modifier.height(40.dp))
+
+            CollectionHeader(
+                headerText = stringResource(R.string.collections_header),
+            )
+
+            Spacer(Modifier.height(10.dp))
+        }
+    }
+}
+
+@Composable
+fun CollectionHeader(
+    headerText: String
+) {
+    FirefoxTheme {
```

Comment:
the FirefoxTheme isn't necessary since `ComposeViewHolder` wraps the `Content` call with it.